kickstart/pack/h4113/hybris

```diff
@@ -142,21 +142,27 @@ FILES_TO_COPY="*.urls"
 if [ "@DEVICEMODEL@" = "h3113" ]; then
   sed -e "s/\@DEVICES\@/\-e \"H3113\" -e \"H3123\" -e \"H3133\"/g" -i flash.sh
   sed -e "s/\@DEVICES\@/:: H3113\r\n:: H3123\r\n:: H3133/g" -i flash-on-windows.bat
+  sed -e "s/\@DEVICES_FINDSTR\@/H3123 H3133/g" -i flash-on-windows.bat    
```

Comment:
Is there any reason to add a second sed command for windows script? You could just replace the existing one as that is not really used for anything just to add a comment to the script.

Reply:
Thanks for the suggestion. I've changed the layout of the which contained @DEVICES@ to do this.